templates/tease-author.twig

```diff
@@ -3,6 +3,7 @@
 		class="d-flex search-result-item-image"
 		src="{{ post.thumbnail.src('thumbnail') }}"
 		alt="{{ fn('esc_attr', post.thumbnail.alt|default( post.title ) )|raw }}"
+		title="{{ fn('esc_attr', post.thumbnail.alt|default( post.title ) )|raw }}"
```

Comment:
This logic is a bit complex and with this change would occur 6 times in 3 files, maybe it's better to write a function for it, or alternative maybe assign it as a template variable? The only difference is [this other one uses `post.post_title`](https://github.com/greenpeace/planet4-master-theme/pull/1395/files#diff-f721ebbf1a207a8e1119c863660a9ec1cac8e82c0e5dcbccebbef1721b41807bR26), but maybe all of them could use `post.title`.

Reply:
For now use a template variable :)